solc/CommandLineInterface.cpp

```diff
@@ -107,6 +107,18 @@ static void version()
 	exit(0);
 }
 
+string jsonPrettyPrint(Json::Value const& input)
```

Comment:
You are right, commonIO only contains actual filesystem routines. You can also add a new file `Json.h`, `JsonFormatter.h`, `JsonIO.h` or something like that.